### DISCUSSION TITLE
Make controls in custom window title transparent for window move operation

### DISCUSSION THREAD

**Discussion by lostmsu** [AvaloniaUI/Avalonia#8842] [Q&A] [answered]
Related to https://github.com/AvaloniaUI/Avalonia/discussions/6013, but I would like "move by the window title" work only on certain parts of the title (e.g. window should be draggable by a `TextBlock` up there, but not by a clickable `MenuItem`). Could I perhaps set hit testing somehow on the controls I need to be transparent for the mouse <-> title interaction?

**Comment by kekekeks** [comment, marked as answer]
IsHitTestVisible="False"

**Reply by lostmsu** [reply]
That did work on a `TextBlock`
